src/routes/PrimaryLayout/PrimaryLayout.js

```diff
@@ -206,8 +211,11 @@ export default class PrimaryLayout extends Component {
         </div>
         <div styleName={cx('detail', { hidden: !hasDetail })} id={DETAIL_COLUMN_ID}>
           <Switch>
-            {detailRoutes.map(({ path, component }) =>
-              <Route path={path} component={component} key={path} />)}
+            {detailRoutes.map(({ path, component }) => {
+              return isAboutPath(location.pathname)
```

Comment:
this feels weird, shouldnt you just list the component as CommunityDetail below in the detailRoutes?

Reply:
Need to pass `communityId` to the component `<CommunityDetail>` which didn't seem possible to do by using the structure in `detailRoutes` below. That's because the CommunityDetail component relies on the id rather than the slug and the id is not available in the routeParams that are passed through.